tabs/webAuthn.tsx

```diff
@@ -0,0 +1,200 @@
+import { useCallback, useEffect, useState } from "react"
+import { runtime, type Runtime } from "webextension-polyfill"
+
+import { sendToContentScript } from "@plasmohq/messaging"
+
+import {
+  ContentMethod,
+  type ContentRequestArguments
+} from "~packages/account/PasskeyAccount/passkeyOwnerWebAuthn/content/method"
+import { PortName } from "~packages/account/PasskeyAccount/passkeyOwnerWebAuthn/tabs/port"
+import json from "~packages/util/json"
+import { objectFromUrlParams } from "~packages/util/url"
+import { createWebAuthn, requestWebAuthn } from "~packages/webAuthn"
+import type {
+  WebAuthnAuthentication,
+  WebAuthnCreation,
+  WebAuthnError,
+  WebAuthnRegistration,
+  WebAuthnRequest
+} from "~packages/webAuthn/typing"
+
+export const WebAuthn = () => {
```

Comment:
Q: Is this page for testing?

Reply:
Yes, this page connects the Dapp sample with the wallet-dapp project. Sample code is [packages/provider/waallet/content/provider.ts](https://github.com/pilagod/waallet-extension/pull/18/files#diff-98ed6e89ec671195fe206f24adf29b5116d28b5a1afb1d8755a1b65769c4403eR25-R35).